common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -165,8 +165,9 @@ class Backend(
         receiptInfo: ReceiptInfo,
         storeAppUserID: String?,
         marketplace: String? = null,
+        sendPricingPhases: Boolean = false,
```

Comment:
perhaps we can use the AppConfig or something instead of `sendPricingPhases` once we set up the `useBC5` parameter